### PR TITLE
add doc to download OpenOrchestra

### PR DESCRIPTION
**Pull request by itkg-bfouche**

### CODE REVIEW

File: app/Resources/doc/dev/draft/OpenOrchestraDownload.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Install OpenOrchestra
+Install open-orchestra with composer:
+
+    ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
+    ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
+
+## Install roles from ansible-galaxy
+Install roles needed to launch the box
+As a prerequisite, update your python modules if required with those two
```
Comment (nicolasThal) [inline app/Resources/doc/dev/draft/OpenOrchestraDownload.md]: a minuscule

File: app/Resources/doc/dev/draft/OpenOrchestraDownload.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    aptitude install python-yaml
+    aptitude install python-jinja2
+
+Then go into openorchestra directory
```
Comment (nicolasThal) [inline app/Resources/doc/dev/draft/OpenOrchestraDownload.md]: then go into the openorchestra directory

File: app/Resources/doc/dev/draft/OpenOrchestraDownload.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+The npm should have also installed the bower component.
+
+Launch the grunt command to generate all assets
+
+    ./node_modules/.bin/grunt
+
+## Load the fixtures
+In the symfony project directory `/var/www/openorchestra`
+
+    php app/console doctrine:mongo:fixture:load -env=prod
```
Comment (nicolasThal) [inline app/Resources/doc/dev/draft/OpenOrchestraDownload.md]: -e=prod ou --env=prod

**Comment by itkg-bfouche** [comment]
@nicolasThal mis à jour

File: app/Resources/doc/dev/draft/OpenOrchestraDownload.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Open Orchestra is composed of two projects, open-orchestra which is the CMS and open-orchestra-front-demo which will display the sites and pages created in the CMS.
+
+## Requirements
+
+To install OpenOrchestra you need:
+
+* Apache2 `sudo aptitude install apache2`
+* PHP `sudo aptitude install php5-dev php5-cli php-pear`
+* Mongo  `sudo pecl install mongo`
```
Comment (nicolasThal) [inline app/Resources/doc/dev/draft/OpenOrchestraDownload.md]: php-mongo

File: app/Resources/doc/dev/draft/OpenOrchestraDownload.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+* PHP extension copy `extension=mongo.so` on your php.ini
```
Comment (nicolasThal) [inline app/Resources/doc/dev/draft/OpenOrchestraDownload.md]: Activate the php mongo extension

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**